openwisp_monitoring/monitoring/tests/__init__.py

```diff
@@ -244,3 +245,19 @@ def _create_chart(self, metric=None, test_data=True, configuration='dummy'):
         c.full_clean()
         c.save()
         return c
+
+    def _read_chart_or_metric(self, obj, *args, **kwargs):
+        if TIMESERIES_DB.get('OPTIONS', {}).get('udp_writes', False):
```

Comment:
avoid repetition of this by creating a function that can be reused across the tests